Fix pricing by country response examples

timcraft opened the pull request:
The keys in the pricing by country [JSON/XML Response examples](https://developer.nexmo.com/api/developer/pricing#response) don't match what's returned by the API (and documented further down under [Keys and Values](https://developer.nexmo.com/api/developer/pricing#keys-and-values) and [Common elements in responses](https://developer.nexmo.com/api/developer/pricing#common-elements-in-responses)).

lornajane commented:
I think this is valuable but I'm struggling to review it because the original source no longer exists, maybe a deleted branch or something similar? We should definitely make some changes as I agree that the docs don't match reality! I think I will need to create new changes since we can't review this, we can only blindly merge.

timcraft commented:
@lornajane Sorry I deleted the fork. I've rebased the change on master, it still applies cleanly:

https://github.com/timcraft/nexmo-developer/tree/fix-per-country-pricing-examples

Can open a fresh pull request if that helps?

timcraft commented:
Closing in favour of #916
